app/api/speakers.py

```diff
@@ -15,13 +15,15 @@
 from app.api.helpers.helpers import model_custom_form, requires_auth
 from app.api.helpers.utils import PAGINATED_MODEL, PaginatedResourceBase, ServiceDAO, \
     PAGE_PARAMS, POST_RESPONSES, PUT_RESPONSES, SERVICE_RESPONSES
+from app.api.helpers.special_fields import SessionStateField
 from app.api.helpers.utils import Resource, ETAG_HEADER_DEFN
 
 api = Namespace('speakers', description='Speakers', path='/')
 
 SPEAKER_SESSION = api.model('SpeakerSession', {
     'id': fields.Integer(),
     'title': fields.String(),
+    'status': SessionStateField(),
```

Comment:
The key should be 'state' here (or use attribute parameter). 
This is because session state is stored as 'state' attribute internally.